### DISCUSSION TITLE
fingerprint js 2.0.0 is out dated?

### DISCUSSION THREAD

**Discussion by YasminNoorjahan** [fingerprintjs/fingerprintjs#819] [Q&A] [answered]
fingerprint js 2.0.0 is out dated or is this not recommended to use?

**Comment by makma** [comment, marked as answer]
Hello @YasminNoorjahan,
Yes, it's an outdated and unsupported version. It doesn't work well with new browsers. We definitely recommend using [this](https://github.com/fingerprintjs/fingerprintjs) one instead.

**Reply by YasminNoorjahan** [reply]
Thanks 👍
